src/components/Search/Search.css

```diff
@@ -0,0 +1,78 @@
+.search {
+  position:relative;
+  margin-right: 20px;
+}
+
+.nav-bar__search {
+  font-size: 22px;
+  fill: black;
+  margin: auto;
+  margin-top: 3px;
+}
+
+.search__button,
+.search__input,
+.search__input--active {
+  height: 42px;
+  width:42px;
+  background-color: rgba(255, 255, 255);
+  border-radius: 100px;
+  outline: none;
+  border: 0;
+}
+
+.search__button {
+  box-shadow: -1px 0px 2px rgba(0, 0, 0, 0.185);
+}
+
+
+.search__button:hover,
+.search__input:hover {
+  transform: translateY(-1px)
+}
+
+.search__button:active {
+  transform: translateY(0px);
+}
+
+.search__button:focus {
+  box-shadow: 0 0 0 2px rgb(65, 25, 156);
+}
+
+
+.search__input,
+.search__input--active {
+  padding-left:30px;
+  z-index:-5;
+  position:absolute;
+  right:0;
+  width: 42px;
+  animation: close 250ms ease-out;
+  transform-origin: left
+}
+
+.search__input--active {
+  animation: open 250ms ease-out;
+  transform-origin: right;
+  width: 250px;
+}
+
+@keyframes open {
+  0% {
+    width:42px
+  }
+  100% {
+    width: 250px
+  }
+}
+
+@keyframes close {
+  0% {
+    width:250px;
+    opacity: 1;
+  }
+  100% {
+    width: 42px;
+    opacity:1
+  }
```

Comment:
Awesome job on the styling for this component! Really well done. It looks great.